integration_tests/cosmoscli.py

```diff
@@ -1079,14 +1079,17 @@ def query_denom_by_contract(self, contract: str):
             )
         )
 
-    def gov_propose_token_mapping_change(
-        self, denom, contract, symbol, decimal, **kwargs
-    ):
-        default_kwargs = {
+    def get_default_kwargs(self):
+        return {
             "gas_prices": DEFAULT_GAS_PRICE,
             "gas": "auto",
             "gas_adjustment": "1.5",
         }
+
+    def gov_propose_token_mapping_change_legacy(
```

Comment:
Is the SDK v46 still supporting the legacy message?
If yes, should we also test it to confirm it is still compatible?

Reply:
yes, why not